docroot/modules/humsci/hs_courses/modules/hs_courses_importer/src/Controller/CoursesController.php

```diff
@@ -126,7 +135,14 @@ protected function setSectionGuids() {
       }
 
       // Build the guid parts.
-      $course_id = $xpath->query('../administrativeInformation/courseId', $course_sections)[0]->textContent;
+      $course_id_element = $xpath->query('../administrativeInformation/courseId', $course_sections);
+      $course_id = 000;
```

Comment:
shouldn't this be a string?

Reply:
yeah i guess the triple zero doesn't do anything huh 👍